### ISSUE TITLE
Dysmorphic features

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6444]
**Preferred term label:**
Dysmorphic features

**Synonyms**
Dysmorphism

**Definition (free text, please give PubMed ID)**
A broad spectrum of defects in physical structures. Such features originate in the first trimester of pregnancy, and their genesis is correlated with genetic susceptibility and environmental conditions during critical time windows. 
DOI: https://doi.org/10.1111/dmcn.12546

**Parent term (use hpo.jax.org/app)**
Abnormality of prenatal development or birth

**Comment by pnrobinson** [comment]
This is probably too broad a term to be useful, and some people use dysmorphism with reference to the face (https://hpo.jax.org/app/browse/term/HP:0001999) and would use the term malformation when speaking of organs. What is the use case?

**Comment by johannaest** [comment]
The patient has dysmorphic features, but not only in the face. That's why I thought a broader term might be useful.

**Comment by pnrobinson** [comment]
It is better to annotate to a list of the precise features in that case. Thanks!
